examples/alexNet/Main.hs

```diff
@@ -83,9 +88,23 @@ an AN{..} input =
     . conv2d'' c1 (4, 4) (2, 2)
     $ input
 
+normalize :: Either String Tensor -> Tensor
+normalize (Right img) = cat 0 [r', g', b']
+    where
+        img' = divScalar (255.0::Float) $ toType D.Float (hwc2chw img)
+        [r, g, b] = I.split (reshape [3, 224, 224] img') 1 0 
+        r' = divScalar (0.229::Float) (subScalar(0.485::Float) r) 
+        g' = divScalar (0.224::Float) (subScalar(0.456::Float) g) 
+        b' = divScalar (0.225::Float) (subScalar(0.406::Float) b) 
```

Comment:
I am not sure if this the best way to normalize. Any suggestions? @junjihashimoto